### PR TITLE
Team updates

### PR DESCRIPTION
**Pull request by pardel**
## Description

Updated entries on the team page.

## Deploy Notes

No changes.

### CODE REVIEW

File: custom/landing_pages/team.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ page:
           content: "<p class='p-large'>Our mission is to build a world-class open source documentation platform to help developers build connected products.</p>"
       - type: html
         html:
-          content: "<p class='p-large'>Here are some of the people behind Nexmo Developer. Oh, and we're hiring for <a href='#join'>the developer relations team</a> and <a href='/careers'>other teams at Vonage</a>."
+          content: "<p class='p-large'>Here are some of the people behind Vonage Developer. Oh, and we're hiring for <a href='#join'>the developer relations team</a> and <a href='/careers'>other teams at Vonage</a>."
       - type: html
```
Comment (superchilled) [inline custom/landing_pages/team.yml:21]: The `#join` fragment identifier doesn't resolve to a particular element on the page. Although it also didn't prior to this PR, it's probably worth fixing as part of these changes.

File: config/team.yml [hunk 2]
```diff
@@ -1,50 +1,41 @@
...
-    role: 'Community Manager'
+    twitter: 'pardel'
   - name: 'Dwane Hemmings'
     email: 'dwane.hemmings@vonage.com'
-    twitter: 'dwaneio'
+    role: "JavaScript Developer Advocate"
+    twitter: 'lifelongdev'
   - name: 'Nick Iati'
     email: 'nick.iati@vonage.com'
     role: 'Technical Writer'
```
Comment (superchilled) [inline config/team.yml:23]: Should Nick be listed as part of the Dev Rel team? If so, do we also want to add Amy and the rest of her team as part of these changes?

**Review by superchilled — CHANGES_REQUESTED** [review]
Looks good overall, though I've noted a couple of questions/ observations.

File: config/team.yml (hunk 2 above)
Comment (pardel) [reply config/team.yml]: Good catch! Removed now.

File: custom/landing_pages/team.yml (hunk 1 above)
Comment (pardel) [reply custom/landing_pages/team.yml:21]: Good point - that should have been an anchor to the jobs below. There is another Jira ticket to update other elements of that page so will include it there.

**Review by superchilled — APPROVED** [review]
Looks good, Paul.

**Pull request merged**